bakery/src/cli/execute.js

```diff
@@ -11,6 +11,23 @@ tmp.setGracefulCleanup()
 
 const sleep = ms => new Promise(resolve => setTimeout(resolve, ms))
 
+const checkDockerMemoryLimit = () => {
+  const dockerSettingsPath = path.join(process.env.HOME, 'Library/Group Containers/group.com.docker/settings.json')
+  if (fs.existsSync(dockerSettingsPath)) {
+    const settings = JSON.parse(fs.readFileSync(dockerSettingsPath))
+    if (!(settings.memoryMiB > 4096)) {
+      console.warn(`
```

Comment:
you can use dedent so you don't break indendation levels with the template string